doc/classes/VisualShape2D.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="VisualShape2D" inherits="Node2D" keywords="square, rectangle, circle, oval, triangle, capsule, hexagon" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		2D shape-drawing node.
+	</brief_description>
+	<description>
+		VisualShape2D is a node for drawing common geometric shapes like rectangles, circles, triangles, and capsules. For more advanced capabilities, see [Polygon2D], [Sprite2D], or [MeshInstance2D]. For custom drawing, see [method CanvasItem._draw] and the [CanvasItem] draw methods.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="get_points" qualifiers="const">
+			<return type="PackedVector2Array" />
+			<description>
+				Returns a list of points representing the vertices of the polygon.
+			</description>
+		</method>
+		<method name="get_uvs" qualifiers="const">
+			<return type="PackedVector2Array" />
+			<description>
+				Returns a list of UV values for each vertex of the polygon.
+			</description>
+		</method>
+	</methods>
+	<members>
+		<member name="antialiased" type="bool" setter="set_antialiased" getter="is_antialiased" default="false">
+			If [code]true[/code], the shape will be antialiased. UVs are not supported on antialiased edges. Scaling may affect antialiasing quality.
+			[b]Note:[/b] This may cause unintentional artifacts when using transparency. Consider adding this to a [CanvasGroup] and using its [member CanvasItem.self_modulate] property for transparency. Alternatively, consider using MSAA ([member ProjectSettings.rendering/anti_aliasing/quality/msaa_2d]).
+		</member>
+		<member name="color" type="Color" setter="set_color" getter="get_color" default="Color(1, 1, 1, 1)">
+			The color of the shape.
+		</member>
+		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset" default="Vector2(0, 0)">
+			The offset amount of the shape in pixels. An offset of [code]0,0[/code] will center the shape. This can be used to move the shape without changing the pivot point.
+		</member>
+		<member name="outline_width" type="float" setter="set_outline_width" getter="get_outline_width" default="0.0">
+			If greater than [code]0[/code], the shape uses an outline instead of being filled. This is the outline width of the shape in pixels. UVs are not supported on outlines.
+		</member>
+		<member name="resolution" type="int" setter="set_resolution" getter="get_resolution" default="64">
+			The resolution to use when the [member shape_type] is [constant SHAPE_CIRCLE] or [constant SHAPE_CAPSULE]. The resolution is used as the number of points in the shape. Large values may affect performance.
```

Comment:
```suggestion
			The resolution to use when the [member shape_type] is [constant SHAPE_CIRCLE] or [constant SHAPE_CAPSULE]. This value is the number of points in the shape. Higher values look smoother but may negatively affect performance.
```
or
```suggestion
			The number of points in the shape if the shape is [constant SHAPE_CIRCLE] or [constant SHAPE_CAPSULE]. Higher values look smoother but may negatively affect performance.
```
Or similar.
Bit of a note: a similar property in StyleBoxFlat is called `corner_detail` and refers to the points as "vertices".

Reply:
I think I'll go with the first but say "This value determines the number of points in the shape", since in capsules its not actually the number of points, its rounded up to the next even number, plus two. (since it needs to be even and the flat sides add some). I'll mention it in the capsule description too.

I could use vertices instead of point here, though I have a method `get_points`. Polygon2D seems to use them interchangeably.

Reply:
"The resolution applied when the [member shape_type] is [constant SHAPE_CIRCLE] or [constant SHAPE_CAPSULE]. This value determines the number of points in the shape. Higher values appear smoother but may negatively affect performance."
My 2 cents, I'd swap out "to use" with applied and "appear" instead of "look".